archunit/src/test/java/com/tngtech/archunit/core/importer/ImportOptionsTest.java

```diff
@@ -186,4 +214,31 @@ FolderPattern expectTestFolder() {
             return new FolderPattern(folders, false);
         }
     }
+
+    private static class StubLocation extends Location {
```

Comment:
As mentioned I think we can do without this :slightly_smiling_face:

Reply:
this class is no longer with us.
fixed in https://github.com/TNG/ArchUnit/pull/794/commits/90c126f6acba68e0602e316c5d576001ee5f20ce